src/bindings/python/src/pyopenvino/graph/passes/transformations.cpp

```diff
@@ -20,6 +20,19 @@
 #include "pyopenvino/utils/utils.hpp"
 
 namespace py = pybind11;
+using Version = ov::pass::Serialize::Version;
+
+
+inline Version convert_to_version(const std::string& version) {
```

Comment:
like placeholder.
Have the same function on `manager.cpp` -- move to `utils.cpp`.

Reply:
done